upgrade-tidb-using-tiup.md

```diff
@@ -1,192 +1,177 @@
 ---
 title: 使用 TiUP 升级 TiDB
-aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/using-tiup/']
+aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/using-tiup/','/docs-cn/dev/upgrade-tidb-using-tiup-offline/', '/zh/tidb/dev/upgrade-tidb-using-tiup-offline']
 ---
 
 # 使用 TiUP 升级 TiDB
 
-本文档适用于使用 TiUP 从 TiDB 3.0 或 3.1 版本升级至 TiDB 4.0 版本，以及从 4.0 版本升级至后续版本。
+本文档适用于使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.0 版本，以及从 5.0 版本升级至后续修订版本。
 
-如果原集群使用 TiDB Ansible 部署，TiUP 也支持将 TiDB Ansible 配置导入，并完成升级。
+如果原集群是 3.0 或 3.1 或更老的版本，需要先升级到 4.0 后再升级到 5.0，不可跨主版本升级。
+
+> **注意：**
+>
+> 从 TiDB 4.0 起，PingCAP 不再提供 TiDB Ansible 的支持。从 5.0 起，不再提供使用 TiDB Ansible 升级集群的文档。如需阅读使用 TiDB Ansible 升级 TiDB 集群的文档，可参阅 [4.0 版使用 TiDB Ansible 升级 TiDB](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-ansible)。
```

Comment:
@yikeke @King-Dylan I don't think this note is necessary here. As Morgan suggested, we should not invite users of a newer version to the doc of an older version. See https://github.com/pingcap/docs/issues/4529. In fact, all users who attempt to read https://docs.pingcap.com/zh/tidb/dev/upgrade-tidb-using-ansible will be redirected to https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-ansible. See https://github.com/pingcap/docs-cn/pull/5682.